po/da.po

```diff
@@ -6,8 +6,8 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: systemd master\n"
-"Report-Msgid-Bugs-To: https://github.com/systemd/systemd/issues\n"
```

Comment:
I believe this should be kept this way. Any way of putting this in the generated POT?